app/api/models/genome.py

```diff
@@ -51,8 +51,8 @@ class GenomeDetails(BaseModel):
     assembly_level: str = Field(alias=AliasPath('attributesInfo', 'assemblyLevel'))
     assembly_date: str = Field(alias=AliasPath('attributesInfo', 'assemblyDate'), default=None)
     annotation_provider: AnnotationProvider = None
-    annotation_method: str = Field(alias='genebuildMethod', default=None)
```

Comment:
@azangru @kamaldodiya I added the mappings to this PR because these are small updates.